src/sagemaker/jumpstart/types.py

```diff
@@ -736,6 +736,334 @@ def _get_regional_property(
         return alias_value
 
 
+class JumpStartBenchmarkStat(JumpStartDataHolderType):
+    """Data class JumpStart benchmark stats."""
+
+    __slots__ = ["benchmark_name", "instance_type", "latency_ms"]
```

Comment:
can you base the benchmark stats off the field in this table? https://quip-amazon.com/NamhAEZUkUFz/QuickSilver-Python-SDK-Demo#temp:C:ZWOc84db6cd2c4e43c8b5661cb88 i updated the mock metadata